### DISCUSSION TITLE
overriding pagination on QTable removes total records and page number

### DISCUSSION THREAD

**Discussion by jwilburn** [quasarframework/quasar#9847] [General - Components / Directives / etc] [answered]
I am looking to override the default pagination on QTable so I can customize the bottom slot to add buttons but after looking at the documentation for `QTable` for `pagination` and `pagination slot`, it appears once you override it, the table no longer shows the total record count or current page (see attached screenshot). Is there a way to add this back when overriding the `pagination` slot?

![image](https://user-images.githubusercontent.com/5719336/123651465-1d15f300-d7f1-11eb-9c23-86d32b21d3df.png)

**Comment by metalsadman** [comment, marked as answer]
overriding slot, means you will have to supply your own pagination component, those info you are trying to display should be available as a scoped property on the slot you are overriding ie. `<template #pagination="props"...>` check api card what properties are exposed in the `props`.

**Reply by jwilburn** [reply]
appreciate the feedback. must have scrolled too fast or spaced out reading the docs.
